stable/cert-manager/templates/NOTES.txt

```diff
@@ -6,10 +6,10 @@ or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).
 More information on the different types of issuers and how to configure them
 can be found in our documentation:
 
-https://cert-manager.readthedocs.io/en/latest/reference/issuers.html
+http://cert-manager.readthedocs.io/en/latest/reference/issuers.html
 
 For information on how to configure cert-manager to automatically provision
 Certificates for Ingress resources, take a look at the `ingress-shim`
 documentation:
 
-https://cert-manager.readthedocs.io/en/latest/reference/ingress-shim.html
+http://cert-manager.readthedocs.io/en/latest/reference/ingress-shim.html
```

Comment:
😬 just spotted this!

I'll make a follow up PR upstream to revert this back.

Reply:
You should fix it in this PR.